app/src/main/java/org/fossasia/susi/ai/adapters/recycleradapters/ChatFeedRecyclerAdapter.java

```diff
@@ -1093,4 +1083,131 @@ private void shareMessage(String message) {
             currContext.startActivity(sendIntent);
         }
     }
+
+    private class SearchResultsAdapter extends RecyclerView.Adapter<SearchResultHolder> {
```

Comment:
Why is this an inner class?
This should be kept in a separate class, any special reason as to why you made this an inner class?